src/NzbDrone.Integration.Test/NzbDrone.Integration.Test.csproj

```diff
@@ -116,13 +116,10 @@
     <Compile Include="IndexHtmlIntegrationFixture.cs" />
     <Compile Include="IntegrationTest.cs" />
     <Compile Include="IntegrationTestDirectoryInfo.cs" />
-    <Compile Include="NamingConfigTests.cs" />
```

Comment:
NamingConfigTests.cs(50,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
NamingConfigTests.cs(63,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
NamingConfigTests.cs(76,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
NamingConfigTests.cs(89,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
NamingConfigTests.cs(101,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
NamingConfigTests.cs(113,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
NamingConfigTests.cs(124,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
RootFolderIntegrationTest.cs(52,26): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'
SeriesIntegrationTest.cs(26,20): error CS0121: The call is ambiguous between the following methods or properties: `FluentAssertions.AssertionExtensions.Should<dynamic>(this System.Collections.Generic.IEnumerable<dynamic>)' and `FluentAssertions.AssertionExtensions.Should(this System.Collections.Generic.IEnumerable<string>)'